### PR TITLE
networkmanager-sstp: init at unstable-2020-04-20, bump sstp from 1.0.12 to 1.0.13

### PR DESCRIPTION
**Pull request by flokli**
###### Motivation for this change
Connecting to SSTP VPNs without manually working with sstpc.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/network-manager/sstp/default.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchFromGitHub, substituteAll, autoreconfHook, sstp, ppp, file, intltool, pkgconfig, networkmanager, libsecret, gtk3
+, withGnome ? true, gnome3, fetchpatch, libnma, glib }:
```
Comment (lovesegfault) [inline pkgs/tools/networking/network-manager/sstp/default.nix:2]: Wish this was split over a few lines

**Review by lovesegfault — APPROVED** [review]
Left a minor nit.

I wish this was made optional, but perhaps that has to happen whenever we make all of NM's plugins optional

c.f. https://github.com/NixOS/nixpkgs/pull/84433

File: pkgs/tools/networking/network-manager/sstp/default.nix (hunk 1 above)
Comment (flokli) [reply pkgs/tools/networking/network-manager/sstp/default.nix]: nit adressed.

**Comment by flokli** [comment]
Okay, let's get this in, and work on making it optional in https://github.com/NixOS/nixpkgs/pull/84433, together with all the other plugins.

**Pull request merged**